package.json

```diff
@@ -2,25 +2,33 @@
   "name": "eslint-visitor-keys",
   "version": "2.1.0",
   "description": "Constants and utilities about visitor keys to traverse AST.",
+  "type": "module",
   "main": "lib/index.js",
+  "exports": {
+    "import": "./lib/index.js",
+    "require": "./dist/eslint-visitor-keys.cjs"
+  },
   "files": [
     "lib"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
   },
   "devDependencies": {
-    "eslint": "^4.7.2",
-    "eslint-config-eslint": "^4.0.0",
+    "c8": "^7.7.3",
+    "eslint": "^7.29.0",
+    "eslint-config-eslint": "^7.0.0",
     "eslint-release": "^3.1.2",
-    "mocha": "^3.5.3",
-    "nyc": "^11.2.1",
-    "opener": "^1.4.3"
+    "mocha": "^9.0.1",
+    "opener": "^1.5.2",
+    "rollup": "^2.52.1"
   },
   "scripts": {
+    "prepare": "npm run build",
+    "build": "rollup -c",
```

Comment:
what is the `-c` used for?

Reply:
If you don't supply the `-c`, the command just gives a help screen. If you put something after the flag, you can indicate a specific config file, but otherwise it defaults to `rollup.config.js` which is what I've done here.

Reply:
well, I see! I just thought it will auto load the config like eslint. :)